core/src/modules/configWriter.ts

```diff
@@ -209,7 +222,8 @@ export namespace ConfigWriter {
     const configFilePath = path.join(configDir, filePath);
     const dir = path.dirname(configFilePath);
     if (!fs.existsSync(dir)) fs.mkdirSync(dir, { recursive: true });
-    const stringifyFilter = (k, v) => (v === null ? undefined : v);
+    const stringifyFilter = (k: string, v: string) =>
+      v === null ? undefined : v;
```

Comment:
v should be unknown, k looks unused.
```suggestion
    const stringifyFilter = (_: string, v: unknown =>
      v === null ? undefined : v;
```